README.md

```diff
@@ -84,6 +84,29 @@ Make changes to the PagerDuty provider and post a pull request for review.
    ```
 4. See `api_url_override` from Terraform docs for [PagerDuty Provider](https://registry.terraform.io/providers/PagerDuty/pagerduty/latest/docs#argument-reference) to set a custom proxy endpoint as PagerDuty client api url overriding service_region setup.
 
+## Test a specific version of the go-pagerduty API client
+
+Modify the `go.mod` file by either of these 2 methods:
+```
+$ go mod edit -replace github.com/heimweh/go-pagerduty=/PATH/TO/LOCAL/github.com/<USERNAME>/<REPO>
+```
+
+Or update the file directly:
+```
+replace github.com/heimweh/go-pagerduty => /PATH/TO/LOCAL/go-pagerduty
+```
+
+Ensure that the vendor package is updated with any changes:
+```
+$ export GOFLAGS="-mod=mod"
+```
+Or:
+```
+$ go mod vendor
+```
+
+**!WARNING!** Do not commit the changes to `go.mod`
```

Comment:
This Warning could result misguiding because while someone is working in an update for the Provider that relies on an also new and unreleased version of `go-pagerduty` is indeed encouraged to commit the replacement for `go-pagerduty` in `go.mod`, and at the end when the dependant `go-pagerduty` version get released (merged into `master`), the the replacement for `github.com/heimweh/go-pagerduty` can be dropped and updated through `go get`, at the end the `vendor` files archive will be the same, but the commit hash pointing to `go-pagerduty` will be the only one changed. In fact, that the best way to avoid failing tests on this repository because of outdated dependencies.

Reply:
Sounds good, I'll just remove this.